user-guide/Basic_Functionality/Visio/making_shapes_display_info/Making_a_shape_display_information_about_the_item_it_is_linked_to.md

```diff
@@ -186,6 +186,8 @@ Add a shape data field of type **Info** to the shape, and set its value to one o
 
   - FUNCTION NAME
 
+  - IN USE: Displays "true" or "false" depending on whether the resource is being used in any bookings (supported from DataMiner 10.3.0/10.2.3 onwards).
```

Comment:
Same remark can be made here regarding updating and performance. Small difference: this will only be updated when the resource is changed, since we are not referring to the resource by GUID.